### DISCUSSION TITLE
Filament and logs table

### DISCUSSION THREAD

**Discussion by AleksandarNeustadt** [filamentphp/filament#7005] [General]
Hello.

I have a table called "categories" and a table called "logs".

I successfully insert data into the "categories" table using Filament. After that insertion, using a model (with some code in the model), I also insert the data into the "logs" table.

The "logs" table has the following columns:

- id
- user_id (who created, updated, edited, or deleted the data)
- table_name (which table, in this case, "categories")
- object_id (in this case, it's the category ID, but it could be a post ID, for example)
- task (created, updated, ...)
- created_at
- updated_at

Everything works fine (I populate rows in both tables after creating a category).

However, when I want to display in fillament table who created the category (the user ID is stored in the "logs" table) and who last updated it (also stored in "logs"), I have a problem doing that.

Could you please provide an example and explain where the solution lies?

Thank you in advance.

**Comment by danharrin** [comment]
It isn't really anything to do with Filament, I think? You could use a Laravel observer to record that sort of information

**Reply by AleksandarNeustadt** [reply]
i hope that i need somthing as 

Tables\Columns\TextColumn::make('createdBy.name'),

I hawe in Category model 
```
public function createdBy()
    {
        return $this->belongsTo(User::class, 'created_by');
    }
```

but, rows crated at is empty in fillament

**Reply by danharrin** [reply]
Does your relationship actually work in the rest of the app? Something must be wrong here on your end
